src/infra/request-promise/index.ts

```diff
@@ -1,54 +1,27 @@
-import Bluebird from 'bluebird';
 import request from 'request';
 
 import { EXTERNAL_HTTP_TIMEOUT_MS } from '../../lib/config.js';
 
-type Request = typeof request;
-
 export type RequestResponse = [request.Response, any];
 
-interface PromisifiedRequest extends Request {
-	getAsync(
```

Comment:
I'm going to miss some of there :cry:

Reply:
In practice we barely use them and they can be replaced with `requestAsync({ method, ...opts })